index.html

```diff
@@ -144,9 +144,9 @@ <h3>The List</h3>
                   The application is an assistant to the headman 
                   to mark the presence of students in the class.
                   <br><br>
-                  <a href="https://github.com/move-fast-and-break-things" target="_blank">
-                    Community
-                  </a>
+                  <span class="developers-list">
```

Comment:
Instead of adding all of the "community members" here, can we instead fetch and add developers who contributed (merged at least on commit) to the main branch of each of the project repos?